docs/SQL/development/visual-editor/visual-expression-builder/use-the-visual-expression-builder.md

```diff
@@ -0,0 +1,168 @@
+1. After creating and setting up the initial Aggregate Gem, hover your pointer between two expression rows and click **+** to add a condition. You're given the option to insert another column or an `IF` or `FOR` condition.
+2. Select **IF**. An `IF` condition appears.
+3. Click **Select expression** and select **Configuration Variable**. Search for or click to select `id_threshold` from the populated list.
+4. Click **Select operator** and select the Comparison operator `greater than`.
+5. Click **Select expression** and select **Value**. Enter `50` as the value.
+6. Optional: You can hover your pointer below the express row you just created and click **+** to add another condition. You're now given additional options to insert an `ELSEIF` or `ELSE` condition.
+
+### Build using the Grouping mode
+
+Let's use a Filter Gem to build grouping expressions.
+
+#### Filter example
+
+![Filter Gem using Grouping mode](img/filter-visual-expression-builder.png)
+
+In our Filter example, we want to filter for the following:
+
+- Total expected revenue that `is not null`
+- Total amounts that are greater than `100000`
+- Latest closed quarters that equals `2023Q2` or `2024Q2`
+
+To set up the grouping expressions, follow these steps:
+
+1. After creating the Filer Gem, click **Add condition**. An option to Select expression appears.
+2. Click **Select expression** and select **Column**. Search for or click to select `TOTAL_EXPECTED_REVENUE` from the populated list. An option to Select operator appears.
+3. Click **Select operator** and select the Existence check `is not null`.
+4. Click **+ Add Condition**. A new expression row appears.
+5. Click **Select expression** and select **Column**. Search for or click to select `TOTAL_AMOUNT` from the populated list. An option to Select operator appears.
+6. Click **Select operator** and select the Comparison operator `greater than`. An option to Select expression appears.
+7. Click **Select expression** and select **Value**. Enter `100000` as the value.
+8. Click **Add Group**. A grouped expression row appears.
+9. Click **Select expression** and select **Column**. Search for or click to select `LATEST_CLOSED_QTR` from the populated list. An option to Select operator appears.
+10. Click **Select operator** and select the Comparison operator `equals`. An option to Select expression appears.
+11. Click **Select expression** and select **Value**. Enter `2023Q3` as the value.
+12. Click **+ Add Condition** and repeat steps 9 to 11 to set up the other `OR` condition.
+    :::tip
+    You can have any number of groups and nestings (a group within a group). And you can change the grouping conditions between `AND` and `OR` by clicking on the labels.
+    :::
+
+## Tips
+
+Here are some additional tips to keep in mind when using the Visual Expression Builder:
+
+- The expression dropdowns support search.
+- Each argument of your function can be another expression since you have the same expression options to choose from.
```

Comment:
Each argument of your function `is` another expression

Reply:
Done.